cli/src/main/java/dev/dirs/Directories.java

```diff
@@ -0,0 +1,70 @@
+package dev.dirs;
+
+import picocli.CommandLine;
+import picocli.CommandLine.*;
+
+@Command(name = "directories", mixinStandardHelpOptions = true, version = "20")
+public class Directories implements Runnable {
```

Comment:
Could we name this more like `DirectoriesCli`/`DirectoriesCLI` or something?

Reply:
Sure!